docs/_sass/_main.scss

```diff
@@ -286,7 +295,7 @@ button {
 #content-area {
     display: flex;
     flex-direction: column;
-    background-color: $color-super-dark-green;
+    background-color: $color-appBG;
```

Comment:
If we give the body a correct color (`$color-appBG`) we might not even need this here, but I don't feel strongly.

Reply:
should we remove it if give the body a correct color @shawnborton